util/memory/tracker_test.go

```diff
@@ -15,61 +15,40 @@ package memory
 
 import (
 	"errors"
+	"fmt"
 	"math/rand"
-	"os"
 	"strconv"
 	"strings"
 	"sync"
 	"testing"
 
 	"github.com/cznic/mathutil"
-	. "github.com/pingcap/check"
 	"github.com/pingcap/parser/terror"
 	"github.com/pingcap/tidb/errno"
-	"github.com/pingcap/tidb/util/logutil"
-	"github.com/pingcap/tidb/util/testleak"
+	"github.com/stretchr/testify/require"
 )
 
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	logLevel := os.Getenv("log_level")
-	err := logutil.InitLogger(logutil.NewLogConfig(logLevel, logutil.DefaultLogFormat, "", logutil.EmptyFileLogConfig, false))
-	if err != nil {
-		t.Errorf(err.Error())
-	}
-	TestingT(t)
-}
-
-var _ = Suite(&testSuite{})
-
-type testSuite struct{}
-
-func (s *testSuite) SetUpSuite(c *C)    {}
-func (s *testSuite) TearDownSuite(c *C) {}
-func (s *testSuite) SetUpTest(c *C)     { testleak.BeforeTest() }
-func (s *testSuite) TearDownTest(c *C)  { testleak.AfterTest(c)() }
-
-func (s *testSuite) TestSetLabel(c *C) {
+func TestSetLabel(t *testing.T) {
 	tracker := NewTracker(1, -1)
```

Comment:
i thought you should add `t.Parallel()` here to make tests as parallel as possible.
the same to other methods.

Reply:
IMO no need blank line here

Reply:
> IMO no need blank line here

There are a few tests with blank line after `t.Parallel()`, maybe it is not necessary.